sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1352,8 +1371,9 @@ private static String getExtractDestinationUri(String extractDestinationDir) {
   /////////////////////////////////////////////////////////////////////////////
 
   /**
-   * A {@link PTransform} that writes a {@link PCollection} containing {@link TableRow TableRows}
-   * to a BigQuery table.
+   * A {@link PTransform} that writes a {@link PCollection} containing objects of type {@code T}
```

Comment:
I would reword this approximately as follows:

"A `PTransform` that writes a `PCollection` to a BigQuery table. A Formatting function must be provided to convert each input element into a `TableRow` using `Write#withFormatFunction(SerializableFunction)`"